index.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-sequences */
+const titleInput = document.querySelector('#title-input');
+const authorInput = document.querySelector('#author-input');
+const addButton = document.querySelector('#add-button');
+const bookCollectionContainer = document.querySelector('#book-collection-container');
+const currentTime = document.querySelector('.current-time');
+const addNewLink = document.querySelector('.add-new-link');
+const formSection = document.querySelector('.form-section');
+const listLink = document.querySelector('.list-link');
+const list = document.querySelector('.list');
+const contactLink = document.querySelector('.contact-link');
+const contactInfo = document.querySelector('.contact-info');
+
+class BookLibrary {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  static books = [];
+
+  static getFromLocalStorage() {
+    const getData = localStorage.getItem('bookArray');
+    if (getData) {
+      return JSON.parse(getData);
+    }
+    return [];
+  }
+
+  static getTitleInput() {
+    return titleInput.value.trim();
+  }
+
+  static getAuthorInput() {
+    return authorInput.value.trim();
+  }
+
+  static createBook() {
+    const title = BookLibrary.getTitleInput();
+    const author = BookLibrary.getAuthorInput();
+    if (title && author) {
+      const newBook = { title, author };
+      BookLibrary.books.push(newBook);
+    }
+    titleInput.value = '';
+    authorInput.value = '';
+  }
+
+  static setToLocalStorage() {
+    return localStorage.setItem('bookArray', JSON.stringify(BookLibrary.books));
+  }
+
+  static displayBooks() {
+    BookLibrary.getFromLocalStorage();
```

Comment:
Nice job!
I will assume you are trying to retrieve some data from your local storage on this line.
However, looking at your `getFromLocalStorage` function, it major responsibility is to return data from your local storage. 
You do not seem to holding its return value in a variable - ideally your `books` array.
This could result in your application not working as intended.
Please work on that.